spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/modal/ModalBottomSheet.kt

```diff
@@ -220,20 +255,26 @@ public fun ModalBottomSheet(
                 contentColor = contentColor,
                 elevation = tonalElevation,
             ) {
-                Column(Modifier.fillMaxWidth()) {
+                Box(
```

Comment:
Let's completly remove our copy of M3 Bottomsheet, currently this was not updated and contains bug that are fixe in M3